README.md

```diff
@@ -121,31 +127,68 @@ And start or stop a specific service with:
     $ pebble start <name1> [<name2> ...]
     $ pebble stop  <name1> [<name2> ...]
 
+## API
+
+The documentation for the currently exposed API endpoints can be found in `doc/api.yaml`
+In that folder you can also find a nice looking html file reflecting the contents of the file above.
+The html file can be generated with: `npx redoc-cli bundle doc/api.yaml --output doc/index.html`
+
+## Layer specification
+
+### Type: Layer
+
+| Field         |         Type         | Default | Required? | Description                                                                                     |
+| :------------ | :------------------: | :-----: | :-------: | :---------------------------------------------------------------------------------------------- |
+| `summary`     |       `string`       |  `nil`  |    no     | Short one line summary of the layer                                                             |
```

Comment:
Note that for most of these strings the default is not actually `nil`, but empty string, but here has the same meaning: "not set".

Reply:
Yeh I debated how to represent this - perhaps I'll go to putting `""` in for correctness.